drf_spectacular/utils.py

```diff
@@ -129,6 +130,10 @@ def __init__(
         self.examples = examples or []
         self.exclude = exclude
 
+        if isinstance(response, list):
+            response = [str(code) for code in response]
```

Comment:
@tfranzel mypy is tripping over this one, it doesn't seem to pick up the `isinstance` check. I've got zero experience with mypy, so no idea what's the best way to resolve this.

```
drf_spectacular/openapi.py:1047: error: Unsupported right operand type for in ("Union[bool, List[Union[int, str]]]")
Found 1 error in 1 file (checked 27 source files)
```

Reply:
I'll move that processing, though I'm usually not a fan of leaking that sort of stuff deep into the implementation. Imo the container class is the best place to normalize data types.